src/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.Directory.Move.cs

```diff
@@ -49,6 +49,7 @@ public void Directory_Move_Multiple_From_Watched_To_Unwatched(int filesCount)
         }
 
         [Theory]
+        [PlatformSpecific(~TestPlatforms.FreeBSD)]
```

Comment:
Will this test never work, or it's a bug right now that it doesn't work?  Just wondering if this should be an ActiveIssue instead.

Reply:
I don't know yet. I want to chat with @aik-jahoda as that test seems unstable on OSX as well. Probably for other reasons but we may find way how to make adjustments. I don't know if reordering is artifact of libinotify or not. But with catching and idle timers in MacOS, I suspect that may happen as well.